ui/src/pages/version/components/forms/components/transformer/components/table_outputs/UpiPostprocessOutputCard.js

```diff
@@ -0,0 +1,67 @@
+import {
+  EuiFlexGroup,
+  EuiPanel,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiFieldText,
+  EuiToolTip,
+  EuiText,
+  EuiSpacer,
+  EuiIcon
+
+} from "@elastic/eui";
+import {
+  useOnChangeHandler
+} from "@gojek/mlp-ui";
+import { DraggableHeader } from "../../../DraggableHeader";
+
+export const UpiPostprocessOutputCard = ({ output, onDelete, onChangeHandler, errors = {}, ...props }) => {
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+  return (
+    <EuiPanel>
+      <DraggableHeader
+        onDelete={onDelete}
+        dragHandleProps={props.dragHandleProps}
+      />
+      <EuiSpacer size="s"/>
+      <EuiFlexGroup direction="column" gutterSize="m">
+        <EuiFlexItem>
+          <EuiToolTip content="UPI Postprocess Output will create output in UPI PredictionValueResponse type">
+            <span>
+            <EuiText size="s">
+              <h4>UPI Postprocess Output</h4>
+            </EuiText>
+              
+            </span>
+          </EuiToolTip>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFormRow
+            label={
+              <EuiToolTip content="Prediction Result Table Name corresponding to the prediction rows provided in the request">
```

Comment:
Should be
`Table name that will be used to populate prediction_result_table in the response to client`